README.md

```diff
@@ -22,26 +41,183 @@ allprojects {
 }
 ```
 
-2. Add your dependency to your app's `build.gradle` file:
+### Available Artifiacts
+
+2. Add relevant dependencies to your app's `build.gradle` file:
```

Comment:
I suggest we re-word this to say something instead like, "here are all of the available SDKs that you can install. We suggest you start only with `tracking` and install others later if you need them". Right now with the way the docs are written, customers don't know what SDK does what and what they need.

Reply:
[The docs look like they do a good job of doing this already](https://github.com/customerio/customerio-android/pull/32/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R79) so maybe we don't even need this paragraph?